Automattic/jetpack
# Social: Add support section and text with icon component

This change adds a new Text with Icon component, and places one into the support section. This should only be visible to paying users.

#### Changes proposed in this Pull Request:

* Added component
* Created new support section
* Conditionally show the new component for paying users

#### Jetpack product discussion



1202726932197503-as-1202670259981508

#### Does this pull request change what data or activity we track or use?



No.

#### Testing instructions:






Testing component:
* Component should match [design](jpldgotrSl8JMHjgUXZLZY-fi-5802%3A39131)
* Check full size and small size screen version too

Testing conditional hiding:
* Fire up jetpack and start Jurassic Tube tunneling. Opening up the site, the support component should not be visible.
* Head to https://wordpress.com/checkout/jetpack_social_monthly and select your Jurassic Tube site
* Finishing up the payment, the section should appear on the admin page.
* Deleting the subscription from My Jetpack should hide it again.

<img width="1220" alt="CleanShot 2022-08-15 at 11 11 38@2x" src="https://user-images.githubusercontent.com/36671565/184608669-3a3e97b7-6753-4327-b4c5-4719f436c034.png">
